File: CellsToServers/HierarchicalTilingApp/Measure/DefaultAuxData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HierarchicalTilingApp.Measure
{
    public class DefaultAuxData : IAuxData
    {
        public int SpaceDimension { get; set; }

        public int HistogramResolution { get; set; }

        public int ServerNO { get; set; }

        public int PointNO { get; set; }

        public int KNN { get; set; }

        public Array Histogram { get; set; }
    }
}

File: CellsToServersCpp/HierarchicalTilingAppCpp/Measure/DefaultAuxData.cpp
#include "DefaultAuxData.hpp"

namespace Measure
{
	int DefaultAuxData::getPointNO() const
	{
		return pointNO;
	}
	
	void DefaultAuxData::setPointNO(int inputPointNO)
	{
		this->pointNO = inputPointNO;
	}
}